.github/workflows/release.yaml

```diff
@@ -94,21 +94,28 @@ jobs:
                 --build-arg TAG=$TAG \
                 ${dir}
           done
+
+  scan:
+    name: Scan images
+    runs-on: ubuntu-24.04
+    needs: build
+    env:
+      YAMORY_ACCESS_TOKEN: ${{ secrets.YAMORY_ACCESS_TOKEN }}
+      ubuntu-version: "20.04 22.04 24.04"
+      ubuntu-image: "ubuntu-debug ubuntu-dev"
+
+    steps:
       - name: Install Trivy
         run: |
           curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin
       - name: Scan images
-        env:
-          YAMORY_ACCESS_TOKEN: ${{ secrets.YAMORY_ACCESS_TOKEN }}
         run: |
-          if [ ! -f BUILDS ]; then
-              echo "nothing to scan."
-              exit 0
-          fi
-          TAG=$(cat TAG)
-          BRANCH=$(cat BRANCH)
-          for d in $(cat BUILDS); do
-              echo
-              echo "scanning $d:$TAG ..."
-              YAMORY_IMAGE_IDENTIFIER="${d}:$BRANCH" YAMORY_IMAGE_NAME="${d}:$TAG" bash -c "$(curl -sSf -L https://mw-receiver.yamory.io/image/script/trivy)"
-          done
+            for ubuntu_version in ${{ env.ubuntu-version }}; do
+              cd $ubuntu_version
```

Comment:
[Q]
Does this work?
It may fail from second loop because the second time it run the `cd` command from `${WORK_DIR}/20.04` to `${WORK_DIR}/20.04/22.04`.

Reply:
I found out that it didn't work when I debugged it on my own, so I fixed the way I wrote it 🙏